src/index.css

```diff
@@ -440,6 +440,109 @@ input[type='text']:disabled {
   height: 50px;
 }
 
+.ant-table-content > table {
+  border-spacing: 0px 10px;
+  width: 100%;
+  background-color: var(--bg-color);
+  border: none !important;
+}
+.ant-table {
+  background-color: inherit !important;
+}
+
+.ant-table-body > table {
+  border-spacing: 0px 10px;
+}
+
+.ant-table-tbody > tr {
+  height: 50px;
+}
+
+.ant-table-thead {
+  background-color: #323232;
+  border-width: 1px 0px;
+  border-radius: var(--corner-radius);
+  font-size: 16px;
+  font-weight: 800;
+  height: 50px;
+}
+
+.ant-table-thead > tr > th {
+  font-size: 16px;
+  background-color: #323232 !important;
+  border-bottom: 1px solid #323232 !important;
```

Comment:
do we need `!important` ? can you explain what it's doing?

Reply:
these modifications were only taking place with the important flag